addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -519,6 +519,11 @@
 		{
 			"prefab"		"159"
 		}
+		"425"	//The Family Business
+		{
+			"desp"			"The Family Business: {positive}speed boost on hit, {negative}50% slower reload speed"
+			"attrib"		"737 ; 1 ; 96 ; 1.5"
```

Comment:
Small nitpicking, value `1` should be in float `1.0` to keep it consistent with other weapons/attrib